discord/threads.py

```diff
@@ -575,9 +575,9 @@ async def join(self):
 
         Joins this thread.
 
-        You must have :attr:`~Permissions.send_messages` and :attr:`~Permissions.use_threads`
-        to join a public thread. If the thread is private then :attr:`~Permissions.send_messages`
-        and either :attr:`~Permissions.use_private_threads` or :attr:`~Permissions.manage_messages`
+        You must have :attr:`~Permissions.send_messages_in_threads` to join a public thread.
+        If the thread is private then :attr:`~Permissions.send_messages_in_threads` and
+        either :attr:`~Permissions.create_private_threads` or :attr:`~Permissions.manage_messages`
```

Comment:
I am not 100% certain this is correct. I assume it will be the case, but it isn't explicitly mentioned in the docs PR.

Reply:
Okay sounds like this isn't correct per checking with Discord staff.

<https://discord.com/channels/613425648685547541/859161948184379403/878334827017928784>

Will update when I'm able 👍